code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -273,6 +273,7 @@
 
 	if(groundside_humans > (groundside_xenos * GROUNDSIDE_XENO_MULTIPLIER))
 		SSticker.mode.get_specific_call("Xenomorphs Groundside (Forsaken)", FALSE, FALSE)
+		SEND_GLOBAL_SIGNAL(COMSIG_GLOB_GROUNDSIDE_FORSAKEN_HANDLING)
```

Comment:
This really shouldn't need to be sent more than once on hijack itself. There's likely a better place to send (best guess, wherever it resets the hive core).

Reply:
Just do that on hijack as I suggested